lib/src/flutter_deck_slide.dart

```diff
@@ -2,6 +2,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter_deck/src/flutter_deck.dart';
 import 'package:flutter_deck/src/flutter_deck_configuration.dart';
 import 'package:flutter_deck/src/flutter_deck_speaker_info.dart';
+import 'package:flutter_deck/src/templates/big_fact_slide.dart';
```

Comment:
Export the template from `lib/src/templates/templates.dart` so you wouldn't need this.